nessie/sql_templates/create_boac_schema.template.sql

```diff
@@ -200,15 +200,16 @@ AS (
         AND {redshift_schema_canvas}.submission_dim.workflow_state != 'deleted'
 );
 
-DROP TABLE IF EXISTS {redshift_schema_boac}.user_course_scores;
+DROP TABLE IF EXISTS {redshift_schema_boac}.course_enrollment;
 
-CREATE TABLE {redshift_schema_boac}.user_course_scores
+CREATE TABLE {redshift_schema_boac}.course_enrollment
```

Comment:
I suppose this should be plural based on naming convention. I'll amend to be `course_enrollments`